### PR TITLE
Fix #10022: Use double click to open template from new score

### LINKED ISSUE #10022: [MU4 Issue] Use double click to open template from New Score Wizard (Improvement)
double click

**Additional context**
from Martin:

> I'd also say pressing 'Enter' should do it too.

### PR DESCRIPTION
**Pull request by gunbux**
Resolves: #10022 

- [x] I signed [CLA](https://musescore.org/en/cla)
- [x] I made sure the code in the PR follows [the coding rules](https://github.com/musescore/MuseScore/wiki/CodeGuidelines)
- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
- [x] I made sure the commit message title starts with "fix #424242:" if there is a related issue
- [ ] I created the test (mtest, vtest, script test) to verify the changes I made

### CODE REVIEW

File: src/project/qml/MuseScore/Project/internal/ChooseInstrumentsAndTemplatesPage.qml [hunk 1]
```diff
@@ -151,5 +153,9 @@ Item {
             id: templatePage
             navigationSection: root.navigationSection
         }
+
+        Component.onCompleted: {
+            templatePage.done.connect(root.done)
+        }
```
Comment (cbjeukendrup) [inline src/project/qml/MuseScore/Project/internal/ChooseInstrumentsAndTemplatesPage.qml:159]: It seems this bit can be simplified, by using the `onDone:` signal handler (which is automatically generated from the `done` signal in ChooseTemplatePage.qml):
```suggestion
            id: templatePage
            navigationSection: root.navigationSection
            
            onDone: {
                root.done()
            }
        }
```

File: src/project/qml/MuseScore/Project/internal/ChooseTemplatePage.qml [hunk 2]
```diff
@@ -88,6 +90,11 @@ Item {
                 model.setCurrentTemplate(index)
             }
 
+            onDoubleClicked: {
```
Comment (cbjeukendrup) [inline src/project/qml/MuseScore/Project/internal/ChooseTemplatePage.qml:93]: Please capture the `index` parameter explicitly here: 
```suggestion
            onDoubleClicked: function(index) {
```
Currently, it is not done consistently in the code, but we really should be doing it at least in new code. Why? To save me some work when making MuseScore ready for Qt 6, where capturing parameters is mandatory. See #10108; 16 of the 75 commits there are about this subject.

**Review by cbjeukendrup — COMMENTED** [review]
Thanks for your contribution! Here are some small suggestions for improvements.

File: src/project/qml/MuseScore/Project/internal/ChooseTemplatePage.qml (hunk 2 above)
Comment (gunbux) [reply src/project/qml/MuseScore/Project/internal/ChooseTemplatePage.qml]: Alright, will keep this in mind moving forward

**Review by RomanPudashkin — APPROVED** [review]

**Comment by DmitryArefiev** [comment]
Tested #10022 on Win10, Mac12, LinuxMint 20.2 - PASS    

Thanks!

Please merge into master (@RomanPudashkin )

**Pull request merged**